### ISSUE TITLE
Web app

### ISSUE THREAD

**Issue by ChocolateLoverRaj** [swatteau/sokoban-rs#14]
The most convenient way to get the game would be if it was a web page. No need to worry about installing or updating.

**Comment by hongliang-goudou** [comment]
Yes, it is quite an interesting game. Should make it as a WebAssembly module to some website. :)
